app/commands/user_track/generate_summary.rb

```diff
@@ -2,75 +2,189 @@ class UserTrack
   class GenerateSummary
     include Mandate
 
-    initialize_with :user_track
-    delegate :track, to: :user_track
+    initialize_with :track, :user_track
```

Comment:
`user_track` can be `nil`, so we need to specify the track explicitely. I might end up with two versions of this generator, one specifically for the nil scenario, but this is fine for now